tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -1134,10 +1135,12 @@ SpirvVariable *DeclResultIdMapper::createExternVar(const VarDecl *var) {
     spvBuilder.decorateInputAttachmentIndex(varInstr,
                                             inputAttachment->getIndex(), loc);
 
-  if (isACRWSBuffer) {
-    // For {Append|Consume|RW}StructuredBuffer, we need to always create another
+  if (isACSBuffer) {
```

Comment:
The only apparent practical methods for Append/Consume Buffers is Append() and Consume() which use the counters. In other words, I was not seeing a practical use of of these buffers that do not use the counters. Am I missing something?

Reply:
Oh no, was just wondering if counters were always needed for A/C buffers, since they are not part of the header. Like if some implementation didn't used counters (I am not familiar with those, so maybe the answer is that yes, no use case without counters 😀)

Reply:
I am not aware of any practical use cases of A/C buffers that do not use the counters. Unless you know of one, I don't think these need to be changed.